games/t.yaml

```
@@ -27,6 +27,22 @@
   - >-
     https://img.itch.zone/aW1hZ2UvMTYwMjM5My8xMDU0MjAxNy5wbmc=/original/weI%2FrN.png
 
+- name: Tails Adventure Remake
+  originals:
+  - Tails Adventure
+  type: remake
+  repo: 'https://github.com/mechakotik/tails-adventure'
+  feed: 'https://github.com/mechakotik/tails-adventure/releases.atom'
+  development: active
+  status: playable
```

```suggestion
  status: semi-playable
```
https://github.com/mechakotik/tails-adventure?tab=readme-ov-file#implemented
some levels have not been implemented yet